src/main/java/nextstep/courses/application/SessionService.java

```diff
@@ -14,7 +14,7 @@ public class SessionService {
 
     public void apply(NsUser loginUser, Long sessionId) {
         Session session = sessionRepository.findById(sessionId);
-        List<Student> students = studentRepository.findBySessionId(sessionId);
-        session.apply(students);
+        Student student = studentRepository.findBySessionId(sessionId);
```

Comment:
```suggestion
        Student student = new Student(sessionId, loginuser.getId());
```

현재 로직이 새롭게 수강신청하는 상황이라 DB에서 Student를 조회하는 것이 아니라 새롭게 Student를 추가해야 하지 않을까?

Reply:
loginUser가 특정 session에 새로이 수강신청하여 해당 session의 Student가 생성되는 것이니 조회가 아닌 생성이 되야겠군요. 감사합니다!!